nifi-nar-bundles/nifi-email-bundle/nifi-email-processors/src/main/java/org/apache/nifi/processors/email/ExtractEmailHeaders.java

```diff
@@ -103,6 +103,22 @@
             .defaultValue("x-mailer")
             .build();
 
+    private static final AllowableValue STRICT_ADDRESSING = new AllowableValue("true", "Strict Address Parsing", "Strict email address format will be enforced. FlowFiles will be transfered to the failure relationship if the email address is invalid.");
```

Comment:
Just ran a build with -Pcontrib-check, and these two lines are failing the style guide due to length (max line width is 200 columns).

@btwood can you fix?

Reply:
@kevdoran I've committed the fix. I didn't know about the -Pcontrib-check build. I added that to my build config and got it to pass. Should be good now. Thanks.